content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -23,8 +24,7 @@ and the current limitations.
 
 ## Using device plugins
 
-Kubernetes implements {{< glossary_tooltip text="Device Plugins" term_id="device-plugin" >}}
-to let Pods access specialized hardware features such as GPUs.
+Kubernetes implements device plugins to let Pods access specialized hardware features such as GPUs.
```

Comment:
Is there a reason for this change?

Reply:
In the overview section above we provided a link to the device plugin page when it is referred for the first time so removed that from here.